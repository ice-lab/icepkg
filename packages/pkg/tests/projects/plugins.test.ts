import { runProjectTest } from './helper';

runProjectTest('plugins', [{
  name: 'custom-format',
  config: 'build.config.custom-format.mts',
  snapshot: 'structure'
}])
