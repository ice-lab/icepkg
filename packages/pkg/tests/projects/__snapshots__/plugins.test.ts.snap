// Vitest Snapshot v1

exports[`Run config custom-format > cjs structure 1`] = `null`;

exports[`Run config custom-format > dist structure 1`] = `
{
  "files": [
    {
      "name": "index.asset.js",
    },
  ],
  "name": "dist",
}
`;

exports[`Run config custom-format > es2017 structure 1`] = `
{
  "files": [
    {
      "name": "index.d.ts",
    },
    {
      "name": "index.js",
    },
  ],
  "name": "es2017",
}
`;

exports[`Run config custom-format > esm structure 1`] = `
{
  "files": [
    {
      "name": "index.d.ts",
    },
    {
      "name": "index.js",
    },
  ],
  "name": "esm",
}
`;
